osu.Game.Tests/Visual/TestCaseHitObjectComposer.cs

```diff
@@ -35,12 +35,12 @@ public class TestCaseHitObjectComposer : OsuTestCase
         private DependencyContainer dependencies;
 
         protected override IReadOnlyDependencyContainer CreateLocalDependencies(IReadOnlyDependencyContainer parent)
-            => dependencies = new DependencyContainer(parent);
```

Comment:
might be handy to just expose the `Dependencies` in the base `TestCase` for general use.